test/site_pages_test.rb

```diff
@@ -10,15 +10,18 @@ def app
   def test_index
     get '/'
     assert last_response.ok?
+    assert doc(last_response).at_css('h1')
   end
 
   def test_about
     get '/about'
     assert last_response.ok?
+    assert doc(last_response).at_css('h1')
```

Comment:
what do you think of testing that it is showing specific text in the heading (otherwise how do you know this isn't just redirecting to index?)